src/github/client/github-client-server.test.ts

```diff
@@ -75,15 +81,18 @@ describe("GitHub Client", () => {
 			"installation token"
 		);
 
-		const appTokenHolder = new AppTokenHolder((installationId: InstallationId) => {
-			switch (installationId.githubBaseUrl) {
-				case gheUrl:
-					return fs.readFileSync(envVars.PRIVATE_KEY_PATH, { encoding: "utf8" });
-				default:
-					throw new Error("unknown GitHub instance!");
-			}
+		when(booleanFlag).calledWith(
+			BooleanFlags.GHE_SERVER,
+			expect.anything(),
+			expect.anything()
+		).mockResolvedValue(true);
+
+		mocked(keyLocator).mockImplementation(async()=> {
+			return fs.readFileSync(envVars.PRIVATE_KEY_PATH, { encoding: "utf8" });
```

Comment:
We probably shouldn't be testing while mocking the `keyLocator`.  It's kinda crucial that it works as expected with the client.

Reply:
Done!